src/actions/guides/http_and_routing/routing_and_params.cr

```diff
@@ -130,17 +143,22 @@ class Guides::HttpAndRouting::RoutingAndParams < GuideAction
 
     ### `route`
 
-    For `route`, it will use the first part of the class name as the resource name,
+    [The macros `route` and `nested-route` do still exist, and automatically imply the default route paths,
```

Comment:
`nested-route` should be `nested_route`.

Reply:
oops